<launch>

	<arg name="map_folder" default="." />
	<arg name="input_cloud_topic" default="/camera/depth_registered/points" />

  	<!-- valgrind -tool=callgrind -->
	 <!-- xterm -e cgdb -args -->
	<node launch-prefix="" pkg="rosmrsmap" type="register_map" name="register_map" respawn="false" output="screen" >
		<param name="map_folder" value="$(arg map_folder)" />
		<remap from="~input_cloud" to="$(arg input_cloud_topic)" />
	</node>

	
</launch>
